cm-mlops/script/get-generic-python-lib/customize.py

```diff
@@ -9,7 +9,8 @@ def preprocess(i):
     meta = i['meta']
     automation = i['automation']
     run_script_input = i['run_script_input']
-
+    pip_version = env.get('CM_PIP_VERSION')
```

Comment:
get can return None too. So, we should use second argument as '' before using any string function.